src/components/TasksList/TasksList.tsx

```diff
@@ -39,20 +39,14 @@ const TasksList: React.FC = () => {
   const [deleteTask, { loading: deleteTaskLoading }] = useMutation<Mutation>(
     DELETE_TASK
   );
-  const [updateTaskStatus, { loading: updateTaskLoading }] = useMutation<Mutation>(
-    UPDATE_TASK_STATUS,
+  const [updateTaskStatus, { loading: updateTaskLoading }] = useTaskStatusUpdate(
```

Comment:
I would suggest to remove the `loading` status in this screen for update tasks the UI would get much more clean.
And we already have the `snackBar` that notifies the user when it finishes updating :)